R/reducolor.R

```diff
@@ -1,15 +1,45 @@
-#' Reduce image colors
+#' Reduce image color to either 2 or 8 colors for cartoonized effect
 #'
-#' @param input_path character the file path of the image
+#' @param style integer either 0 or 1
+#'              0 for white and black colors
+#'              1 for 8 colors
+#' @param input_path character the image file path
 #' @param output_path character if not None, the modified image will be saved
-#'                    in the provided folder path
-#' @param ncolor integer number of colors to be reduced to
+#'                    in the provided folder path and name
 #'
 #' @return modified image array
 #' @export
 #'
 #' @examples
-#' redu_color("images/milad.jpg", "images", 4)
-redu_color <- function(input_path, ncolor, output_path=NULL) {
+#' redu_color(0, "tests/milad.jpg", "tests/milad_new.jpg")
```

Comment:
```suggestion
#' redu_color(0, "../milad.jpg", "../milad_new.jpg")
```
Changing the example to be consistent with the test file